### DISCUSSION TITLE
Howto run go scheduler and goroutines within browser?

### DISCUSSION THREAD

**Discussion by unicomp21** [pion/webrtc#1976] [Q&A] [answered]
The pion wasm stuff, is there a way to run goroutines in the browser?  if so, i'm looking for an example/repo.  I've found a few small snippets referencing go_scheduler().  Is it as simple as calling go_scheduler() from request_animation()?

**Comment by Sean-Der** [comment, marked as answer]
Hey @unicomp21 

The Pion WASM implementation really just does FFI and uses the browsers WebRTC implementation. I haven't done a lot of WASM work, but AFAIK Go just runs with a single proc (thread).
